src/explore-education-statistics-admin/src/pages/methodology/MethodologiesPage.tsx

```diff
@@ -52,105 +107,60 @@ const MethodologiesPage = () => {
       </div>
 
       <Tabs id="methodologyTabs">
-        <TabsSection id="manage-methodology" title="Methodologies">
+        <TabsSection
+          id="live-methodologies"
+          title={`Live methodologies ${
+            model && model.liveMethodologies.length
+              ? `(${model.liveMethodologies.length})`
+              : '(0)'
+          }`}
+        >
+          <Link
+            to="/methodologies/create"
+            className="govuk-button govuk-!-margin-bottom-1"
+          >
+            Create new methodology
+          </Link>
           {model && model.liveMethodologies.length ? (
-            <table className="govuk-table">
-              <thead className="govuk-table__head">
-                <tr className="govuk-table__row">
-                  <th scope="col" className="govuk-table__header">
-                    Methodology title
-                  </th>
-                  <th scope="col" className="govuk-table__header">
-                    Status
-                  </th>
-                  <th scope="col" className="govuk-table__header">
-                    Related publications
-                  </th>
-                </tr>
-              </thead>
-              <tbody className="govuk-table__body">
-                {model.liveMethodologies.map(methodology => (
-                  <tr className="govuk-table__row" key={methodology.id}>
-                    <td className="govuk-table__header">
-                      <Link to={contentRoute.generateLink(methodology.id)}>
-                        {methodology.title}
-                      </Link>
-                    </td>
-                    <td className="govuk-table__cell">
-                      <strong className="govuk-tag">
-                        {methodology.status}
-                      </strong>
-                    </td>
-                    <td className="govuk-table__cell">
-                      <ul className="govuk-list">
-                        {methodology.publications.map(publication => (
-                          <li key={publication.id}>{publication.title}</li>
-                        ))}
-                      </ul>
-                    </td>
-                  </tr>
-                ))}
-              </tbody>
-            </table>
+            <MethodologiesTable methodologies={model.liveMethodologies} />
           ) : (
             <div className="govuk-inset-text">
-              There is currently no draft methodology
+              There are currently no live methodologies
```

Comment:
Not a dealbreaker but the empty message logic could be placed inside the `MethodologiesTable` component to reduce duplication.